script/release/kie-upgrade-branch-auxiliary-reps.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash -e
+
+# ***************************************************************************
+#                                                                           *
+# This script branches and upgrades to the next SNAPSHOT version            *
+# the repositories                                                          *
+#   kie-docker-ci-images (https://github.com/kiegroup/kie-docker-ci-images) *
+#   kie-benchmarks (https://github.com/kiegroup/kie-benchmarks)             *
+#   kie-cloud-test (https://github.com/kiegroup/kie-cloud-tests)            *
+# These repositories are not in the repository-list.txt, but an upgrade and *
+# a branching is needed.                                                    *
+#                                                                           *
+# ***************************************************************************
+
+# checks how many files have changed after version upgrade
+linesCount () {
+   nrLines=$(git status -s | wc -l | awk '{ print $1 }')
+   #echo "nrLines= "$nrLines
+   if [ "$nrLines" -gt "$count" ]; then
+     echo "version upgrade of $rep to $newSnapshot was faulty"
+     exit 1
+   fi
+}
+
+
+# ******************** kie-docker-ci-images ************************************
+
+# kie-docker-ci-images
+rep="kie-docker-ci-images"
+git clone git@github.com:kiegroup/kie-docker-ci-images.git
+cd kie-docker-ci-images
+git checkout -b $newBranch master
+git checkout master
+
+# upgrade the versions to next SNAPSHOT
+./scripts/update-versions.sh $newSnapshot -U
+
+# count of pom.xml files with SNAPSHOT
+count=$(grep -rli --exclude-dir="src" "snapshot" | grep pom.xml | wc -l)
```

Comment:
This can be simplified to single grep command, which is more reliable:
`grep snapshot -rli --include=pom.xml --exclude-dir=src --count`